akka-actor-typed/src/main/scala/akka/actor/typed/receptionist/Receptionist.scala

```diff
@@ -297,6 +373,20 @@ object Receptionist extends ExtensionId[Receptionist] {
   def listing[T](key: ServiceKey[T], serviceInstances: java.util.Set[ActorRef[T]]): Listing =
     Listing(key, Set[ActorRef[T]](serviceInstances.asScala.toSeq: _*))
 
+  /**
+   * Java API: Sent by the receptionist, available here for easier testing
+   */
+  def listing[T](
+      key: ServiceKey[T],
+      serviceInstances: java.util.Set[ActorRef[T]],
+      allServiceInstances: java.util.Set[ActorRef[T]],
+      allServiceInstancesChanged: Boolean): Listing =
+    Listing(
+      key,
+      Set[ActorRef[T]](serviceInstances.asScala.toSeq: _*),
+      Set[ActorRef[T]](allServiceInstances.asScala.toSeq: _*),
```

Comment:
better to use `scala.collection.JavaConverters` ?

Reply:
Wasn't sure of the rationale for the original code being like this but thought there was a reason (tm) so that's the copy-pasta rationale